cmd/access/node_builder/staked_access_node_builder.go

```diff
@@ -221,6 +221,7 @@ func (builder *StakedAccessNodeBuilder) Build() (cmd.Node, error) {
 				builder.rpcMetricsEnabled,
 				builder.apiRatelimits,
 				builder.apiBurstlimits,
+				nil,
```

Comment:
this file should no longer exist after the refactor.

Reply:
I planned to add an changelist once the concepts and earlier changelists are approved.

Reply:
This is the final layout: https://github.com/GetElastech/flow-go/commit/33623bb268ecfaedd1680372860ab98e6a35b7b6
```
cmd/access/
cmd/access/main.go
cmd/access/README.md
cmd/access/.gitignore
cmd/access/node_builder
cmd/access/node_builder/access_node_builder.go```